flutter/lib/ui/geometry.dart

```diff
@@ -680,35 +671,102 @@ class Rect {
   String toString() => "Rect.fromLTRB(${left.toStringAsFixed(1)}, ${top.toStringAsFixed(1)}, ${right.toStringAsFixed(1)}, ${bottom.toStringAsFixed(1)})";
 }
 
-// A rounded rectangle with the same radii for all four corners.
+/// A radius for either circular or oval shapes.
+class Radius {
+  /// Constructs a circular radius.
+  const Radius.circular(double radius) : this.oval(radius, radius);
```

Comment:
Radius.circle maybe? "circular" and "oval" aren't the same part of speech so currently it's confusing.